### PR TITLE
Breq 1263 ubuntu migration

### PR DESCRIPTION
**Pull request by kylep**

### CODE REVIEW

File: docs/migrations/ubuntu-conversion.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+volume to enforce the names, and write interface files accordingly. This is the `udev` same
+procedure in Ubuntu as it is in RedHat.
+
+- Interface names are commonly `ens0`, `ens1`, `ens2`, and so on.
+- Prefix is the CIDR prefix. For `255.255.255.0`, the prefix is `24`.
+- Reference: [Consistent network device naming](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/networking_guide/ch-consistent_network_device_naming)
+
+```bash
+# Example of setting a DHCP interface
+voithos migrate rhel set-interface <device> --name <interface name> --dhcp --mac "<mac address>"
```
Comment (waqasnazir03) [inline docs/migrations/ubuntu-conversion.md:82]: rhel --> ubuntu?

File: docs/migrations/ubuntu-conversion.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+# Setting a static interface with default route and DNS settings.
+# --gateway, --dns, and --domain are all optional
+voithos migrate rhel set-interface \
```
Comment (waqasnazir03) [inline docs/migrations/ubuntu-conversion.md:86]: rhel --> ubuntu

File: docs/migrations/ubuntu-conversion.md (hunk 1 above)
Comment (kylep) [reply docs/migrations/ubuntu-conversion.md:82]: Oops, yes rhel --> ubuntu. On it!

**Review by waqasnazir03 — COMMENTED** [review]
Added minor comments.

**Comment by waqasnazir03** [comment]
nvm, they are fixed in the last commit.

**Review by waqasnazir03 — APPROVED** [review]

**Pull request merged**